app.js

```diff
@@ -6,6 +8,26 @@ const signUpForm = document.getElementById('sign-up');
 const signUpEmail = document.getElementById('sign-up-email');
 const signUpPassword = document.getElementById('sign-up-password');
 
+redirectIfLoggedIn();
+
 // Wire up sign in and sign up forms to supabase
+
```

Comment:
Nice work I would also eslint disable console.logs for each file you keep them in so it want give a warning. The warning will not fail the build or eslint but it still is nice not to have any warnings